src/containers/MyNdla/Arena/components/ArenaListItem.tsx

```diff
@@ -63,12 +63,25 @@ const ContentWrapper = styled("div", {
 
 const NumberText = styled(Text, {
   base: {
+    minWidth: "68px",
```

Comment:
Spørsmålet er om vi skal følge designet eller bruke spacing/semantic values 🤔 Jeg syntes dette ser bedre ut enn `3xlarge` hvertfall

Reply:
Jeg er av den mening at vi skal bruke semantic-tokens så langt det lar seg gjøre. Men i dette tilfellet ser jeg egentlig ikke helt grunnen til å sette en bredde i det hele tatt. Kan vi ikke bare gjøre wrapperen til en grid? Så tar begge gjenstandene opp like mye plass.

Reply:
Jeg er enig, trodde opprinnelig at 32-token ble for lite, men ser ut som det skal gå greit likevel!

Apropos grid, så ser jeg ikke hvordan det blir en penere løsning? Problemet er jo ikke at kolonnene på samme rad ikke er like store, det er jo at kolonnene på tvers av radene skal være like store, så sånn jeg ser det ender vi vel opp med å måtte sette noe form for bredde uansett?

Reply:
Ah, skjønner. Jeg tror egentlig dette burde være et designproblem, isåfall. Hedvig skal jo strengt tatt holde seg til semantiske tokens.